pkg/api/v1/types.go

```diff
@@ -684,6 +685,24 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,4,opt,name=readOnly"`
 }
 
+// Represents a Quobyte mount that lasts the lifetime of a pod.
+// Quobyte volumes do not support ownership management or SELinux relabeling.
+type QuobyteVolumeSource struct {
+	// Registry represents a single or multiple Quobyte Registry services
```

Comment:
it's a host port pair (or multiple host port pairs separated by commas". The host can be an IP address or a DNS name and the port must be the Quobyte API Port (default: 7861).